css/main.css

```diff
@@ -188,3 +188,90 @@ ul.grid-list > li:before {
 .search-item.focus-visible li > div {
   @apply bg-gray-800 text-gray-200;
 }
+
+.zustand-home .bg {
+  background-image: url('https://uploads.codesandbox.io/uploads/user/b3e56831-8b98-4fee-b941-0e27f39883ab/DWLR-bg.jpg');
```

Comment:
We're already hosting these images. Is this outdated or should we point to them?

Reply:
oh crap, will update 


Thanks!

Reply:
This seems like its not even being used anymore

Reply:
Deleted here: https://github.com/pmndrs/website/commit/2517d46b094bbc000455bdf9959fe689ef846b6f